Extension/src/LanguageServer/extension.ts

```diff
@@ -474,10 +474,15 @@ function realActivation(): void {
             if (vscodeVersion.isGreaterThan(minimumSupportedVersionForInsidersUpgrades, "insider")) {
                 insiderUpdateEnabled = true;
                 if (settings.updateChannel === 'Default') {
-                    suggestInsidersChannel();
+                    const userVersion: PackageVersion = new PackageVersion(util.packageJson.version);
+                    if (userVersion.suffix === "insiders") {
+                        checkAndApplyUpdate(settings.updateChannel, false);
```

Comment:
Should this be `true` because there was a settings change since the last time we checked?

Reply:
No, this is the "onInit" case where we want the autoUpdate check to not be bypassed.

Reply:
But in this case it would be a "downgrade", so autoUpdate doesn't apply (?)

Reply:
update could mean upgrade or downgrade, so it seems like we should observe autoUpdate still, which is the existing behavior.